public/app/percona/dbaas/components/Kubernetes/Kubernetes.hooks.ts

```diff
@@ -81,10 +92,37 @@ export const useKubernetes = (): [Kubernetes[], DeleteKubernetesAction, AddKuber
   return [kubernetes, deleteKubernetes, addKubernetes, loading];
 };
 
-const toModelList = (response: KubernetesListAPI): Kubernetes[] => (response.kubernetes_clusters ?? []).map(toModel);
+const toModelList = (response: KubernetesListAPI, checkUpdateResponse: CheckOperatorUpdateAPI): Kubernetes[] =>
+  (response.kubernetes_clusters ?? []).map(toModel(checkUpdateResponse));
 
-const toModel = (response: KubernetesAPI): Kubernetes => ({
-  kubernetesClusterName: response.kubernetes_cluster_name,
-  operators: response.operators,
-  status: response.status as KubernetesClusterStatus,
+const toModel = (checkUpdateResponse: CheckOperatorUpdateAPI) => ({
+  kubernetes_cluster_name,
```

Comment:
nit: I would consider aliasing this to camelCase

Reply:
Done!